### ISSUE TITLE
Request for bug fixes in oproxy.py

### ISSUE THREAD

**Issue by YuehuaZhu** [ExplainableML/NonIsotropicProxyDML#2]
In File "./NonIsotropicProxyDML-main/criteria/oproxy.py", line 64 (proxies = self.prep(self.proxies)), in compute_proxyloss, 'Criterion' object has no attribute 'prep'.

Many thanks to the author for providing the source code, there is a bug here,  and I hope you will fix it as soon as possible.

**Comment by Confusezius** [comment]
Thanks for pointing to this bug! It was a remnant from the code base change, it's fixed now :).
`self.prep()` was literally just `torch.nn.functional.normalize(..., dim=-1)` :).
